### DISCUSSION TITLE
How to create a unidirectional elastic support in Robot Structural Analysis?

### DISCUSSION THREAD

**Discussion by dextertuay99** [BHoM/BHoM#1572] [Q&A]
I've tried assigning FixedPositive to Constrain6DOF on the Edge of my Panel. However, the outcome in Robot is not as what I expected. 

I'm expecting both, at the same time:
Rigid: UY+
Elastic: KY (with any value)

However, it only push:
Rigid: none
Elastic: KY

Any help?

**Comment by FraserGreenroyd** [comment]
@peterjamesnugent @al-fisher would either of you be able to advise on this? @Chrisshort92 maybe?

**Comment by peterjamesnugent** [comment]
Hi @dextertuay99, apologies for not reaching out sooner on the issue you raised - I have answered it here: 
https://github.com/BHoM/Robot_Toolkit/issues/530#issuecomment-1853654792
